cmd/juju/model/checkout.go

```diff
@@ -18,71 +18,77 @@ import (
 )
 
 const (
-	switchGenerationSummary = "Switch to the given generation."
-	switchGenerationDoc     = `
-Switch to the given generation, either current or next. 
+	checkoutSummary = "Work on the supplied branch."
+	checkoutDoc     = `
+Makes the supplied branch active, causing changes to charm configuration,
```

Comment:
I'm not sure I like the word active in this context, but I might be nit picking here, it implies that previously it was disabled.

Potential?
> Switch to the supplied branch

Reply:
Changed it.